docs/application/dotnet/samples/ui/overview.md

```diff
@@ -189,56 +189,56 @@ for TD:
 		<tr>
 			<td><img alt="" height="180" src="media/w46_animationsample_nui.png" width="180"/></td>
 			<td>
-			<p><strong>(W) AnimationSample (NUI)</strong> [In progress]</p>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/AnimationsSample" target="_blank"><strong>(W) AnimationSample (NUI)</strong></a></p>
 			<p>This sample application demonstrates how to animate NUI objects using the NUI animation.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="180" src="media/w47imagesample_nui.png" width="180"/></td>
 			<td>
-			<p><strong>(W) ImageSample (NUI)</strong> [In progress]</p>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/ImageSample" target="_blank"><strong>(W) ImageSample (NUI)</strong></a></p>
 			<p>This sample application demonstrates how to load and control images.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="180" src="media/w48scriptlayoutsample_nui.png" width="180"/></td>
 			<td>
-			<p><strong>(W) ScriptLayoutSample (NUI)</strong> [In progress]</p>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/ScriptLayoutSample" target="_blank"><strong>(W) ScriptLayoutSample (NUI)</strong></a></p>
 			<p>This sample application demonstrates how to set the theme of an application using script file.
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="180" src="media/w49texteditorsample_nui.png" width="180"/></td>
 			<td>
-			<p><strong>(W) TextEditorSample (NUI)</strong> [In progress]</p>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/TextEditorSample" target="_blank"><strong>(W) TextEditorSample (NUI)</strong></a></p>
 			<p>This sample application demonstrates how to control the TextEditor properties.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="180" src="media/w50textfieldsample_nui.png" width="180"/></td>
 			<td>
-			<p><strong>(W) TextFieldSample (NUI)</strong> [In progress]</p>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Wearable/TextFieldSample" target="_blank"><strong>(W) TextFieldSample (NUI)</strong></a></p>
 			<p>This sample application demonstrates how to control the TextField properties.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="180" src="media/w51textlabelsample_nui.png" width="180"/></td>
```

Comment:
@sgchoi5 Would you please confirm that w51textlabelsample_nui.png and w50textfieldsample_nui.png are same?
Both images have "Text Field" as the title, and I'm not sure with that.

Reply:
Changed images!